### PR TITLE
feat: Add import private key in controller (wasm,rest)

### LINKED ISSUE #1809: Add import private key in controller (wasm,rest)

### PR DESCRIPTION
**Pull request by fqutishat**
closes #1809

Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

File: pkg/controller/command/kms/models.go [hunk 1]
```diff
@@ -11,6 +11,13 @@ type CreateKeySetRequest struct {
 	KeyType string `json:"keyType,omitempty"`
 }
 
+// ImportPrivateKeyRequest is model for importPrivateKey request.
+type ImportPrivateKeyRequest struct {
```
Comment (troyronda) [inline pkg/controller/command/kms/models.go:15]: should use JWK format.

File: pkg/controller/command/kms/command.go [hunk 2]
```diff
@@ -111,3 +132,52 @@ func (o *Command) CreateKeySet(rw io.Writer, req io.Reader) command.Error {
...
+		logutil.LogDebug(logger, commandName, importPrivateKeyCommandMethod, errEmptyKeyID)
+		return command.NewValidationError(InvalidRequestErrorCode, fmt.Errorf(errEmptyKeyID))
+	}
+
+	var privKey interface{}
+
+	switch r := request.KeyType; {
+	case strings.Contains(r, "ED25519"):
+		privKey = ed25519.PrivateKey(base58.Decode(request.PrivateKey))
+	default:
```
Comment (troyronda) [inline pkg/controller/command/kms/command.go:166]: and P256

File: pkg/controller/rest/kms/operation.go [hunk 3]
```diff
@@ -70,3 +73,13 @@ func (o *Operation) registerHandler() {
...
 	rest.Execute(o.command.CreateKeySet, rw, req.Body)
 }
+
+// ImportPrivateKey swagger:route POST /kms/import kms importPrivateKey
+//
+// Import private key.
+//
+// Responses:
+//    default: genericError
+func (o *Operation) ImportPrivateKey(rw http.ResponseWriter, req *http.Request) {
```
Comment (troyronda) [inline pkg/controller/rest/kms/operation.go:83]: just make this ImportKey

File: test/aries-js-worker/test/kms/kms.js [hunk 4]
```diff
@@ -56,4 +56,17 @@ async function kms(mode) {
             err => done(err)
         )
     })
+
+    it("Alice import private key", function (done) {
+        aries.kms.importPrivateKey({
+            keyType: "ED25519",
+            keyID:"k1",
+            privateKey:"5PNvrZYz5W23MzeMVKFxhvL9SfRVcnT3ZdBHAAHesB4RJcggreQfFNP3LQRUvkwmEEfahQkJaxUxdJwEFxBhg6tp"
```
Comment (troyronda) [inline test/aries-js-worker/test/kms/kms.js:64]: you don’t also need the public key?

File: pkg/controller/command/kms/command.go (hunk 2 above)
Comment (fqutishat) [reply pkg/controller/command/kms/command.go:155]: done

File: pkg/controller/command/kms/models.go (hunk 1 above)
Comment (fqutishat) [reply pkg/controller/command/kms/models.go]: done

File: pkg/controller/command/kms/command.go (hunk 2 above)
Comment (fqutishat) [reply pkg/controller/command/kms/command.go:155]: done

File: pkg/controller/rest/kms/operation.go (hunk 3 above)
Comment (fqutishat) [reply pkg/controller/rest/kms/operation.go]: done

File: test/aries-js-worker/test/kms/kms.js (hunk 4 above)
Comment (fqutishat) [reply test/aries-js-worker/test/kms/kms.js]: done

**Pull request merged**